__tests__/app.test.js

```diff
@@ -192,6 +193,7 @@ describe('GET /api/articles/:article_id', () => {
                     body: "I find this existence challenging",
                     created_at: "2020-07-09T20:11:00.000Z",
                     votes: 100,
+                    comment_count: "11"
                   }
             )
             expect(article.article_id).toBe(1);
```

Comment:
Rather than editing a previous test, could we please implement a new test for this ticket?